tests/test_evolution_engine.py

```
@@ -4,20 +4,314 @@
+    )
+
+
+def test_orchestrator_registers_interval_schedule(
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    backend = DummyWorkflowBackend()
+
+    class _NoopTrainer:
+        def __init__(
+            self, training_config_path: str, output_dir: str
+        ) -> None:  # noqa: D401
```

Ruff (0.13.3)

Unused `noqa` directive (non-enabled: `D401`)

Remove unused `noqa` directive

(RUF100)